packages/ai-jsx/src/lib/anthropic.tsx

```diff
@@ -120,16 +118,16 @@ export async function* AnthropicChatModel(
       .map(async (message) => {
         switch (message.tag) {
           case UserMessage:
-            return `${AnthropicSDK.HUMAN_PROMPT}:${message.props.name ? ` (${message.props.name})` : ''} ${await render(
+            return `${AnthropicSDK.HUMAN_PROMPT}${message.props.name ? ` (${message.props.name})` : ''} ${await render(
```

Comment:
Do we still want the AnthropicSDK references here?

Reply:
I checked, and they're just generic `Human: ` and `Assistant: ` strings. I think it's fine + better to just hardcode it than having this function take them as arguments.

Reply:
it seems a bit weird for Llama to use these references though. Maybe this goes away when we move to a Llama chat model?

Reply:
Yep – I just ripped this out.